adwords_client/adwords_api/operations/campaign.py

```diff
@@ -6,7 +6,7 @@ def campaign_operation(campaign_id: 'Long' = None,
                        budget_id: 'Long' = None,
                        status: 'String' = 'PAUSED',
                        advertising_channel: 'String' = 'SEARCH',
-                       network_setting: 'String[]' = None,
+                       network_setting: 'StringList' = None,
```

Comment:
We are copying from Google documentation. This way we keep the compatibility maintenance low.

Please change it back.